src/pybind/mgr/orchestrator.py

```diff
@@ -183,6 +183,18 @@ def describe_service(self, service_type=None, service_id=None, node_name=None):
         """
         raise NotImplementedError()
 
+    def service_operation(self, operation, service_type, service_id=None):
```

Comment:
Is there a better or more precise word for "`operation`", something that excludes adding and removing services?

Reply:
I don't know, but I'm totally open to suggestions ("operation" was the least worst thing I could think of :-/)

Reply:
@sebastian-philipp, what about "action" instead?  It's a bit narrower in scope.